guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md

```diff
@@ -69,7 +69,7 @@ Attribute | Description
 --- | ---
 `name` | The name of the table
 `engine` | SQL engine. This value must be `innodb` or `memory`.
-`resource` | The database shard on which to install the table. This value must be `default`, `quote`, or `sales`.
+`resource` | The database shard on which to install the table. This value must be `default`, `checkout`, or `sales`.
```

Comment:
Correct, referring to: https://github.com/magento/magento2/blob/6b69fa145def326c4eb8faa2972628e1ca3f9f86/lib/internal/Magento/Framework/Setup/Declaration/Schema/etc/schema.xsd#L99